docs/network_disruption/prio.md

```diff
@@ -205,8 +207,10 @@ spec:
     port: 80
     protocol: tcp
     http:
-        method: get <-- eBPF filter
-        path: /test <-- eBPF filter
+        methods:
```

Comment:
I have no objections to choosing this approach, but why didn't we decide to let users specify methods _per_ path? Does it create too many tc filters?

Reply:
I have tried to implement this kind of feature, but it was really hard (memory/complexity) to pair paths with methods so after 1 week I have abandoned this idea. 😞

Reply:
That's a good enough reason for me.